test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -99,6 +99,7 @@ def run_smdebug_test(
     framework = get_framework_from_image_uri(image_uri)
     container_test_local_dir = os.path.join("$HOME", "container_tests")
     ec2_connection.run(f"$(aws ecr get-login --no-include-email --region {region})", hide=True)
+    ec2_connection.run(f"docker pull -q {image_uri}")
```

Comment:
How was this working before without pulling the docker image?

Reply:
`docker run` automatically pulls an image as long as it can find the image on the repository, and fails if it can't pull it.

I've been seeing failed tests that timeout on the `docker run` command for tests which seem to take about 4-5 minutes, which stopped happening when I inserted this forced `docker pull` before the `docker run` into the test code.